Spawn Eggs, Music Discs, Dyes, and more!

APickledWalrus opened the pull request:
I updated misc.sk and completed some of the TODOs.

+ Added spawn eggs (1.13 + 1.12 & older)
+ Added music discs (1.13 + 1.12 & older)
+ Added fish buckets to buckets section
+ Moved horse armor from combat.sk to misc.sk
+ Fixed iron nuggets and added "any nugget"
+ Various other edits

TheBentoBox commented on misc.sk:
```diff
@@ -75,10 +75,232 @@ plants:
+	any bucket = empty bucket, water bucket, lava bucket, milk
+new buckets:
+	minecraft version = 1.13 or newer
+	tropical fish bucket¦s = minecraft:clownfish_bucket
```
It looks to me, [from the wiki](https://minecraft.gamepedia.com/Bucket_of_Fish#ID), that the ID for this is `tropical_fish_bucket`, not `clownfish_bucket`.

TheBentoBox commented on misc.sk:
```diff
@@ -75,10 +75,232 @@ plants:
+	cod bucket¦s = minecraft:cod_bucket
+	pufferfish bucket¦s = minecraft:pufferfish_bucket
+	salmon bucket¦s = minecraft:salmon_bucket
+	any bucket = empty bucket, water bucket, lava bucket, milk, tropical fish bucket, cod bucket, pufferfish bucket, salmon bucket
```
I'd recommend adding a separate `any fish bucket` alias.

APickledWalrus commented:
The tropical fish bucket should be right now, and I also added the any fish bucket list you recommended. Thanks for the quick reply :)

TheBentoBox commented:
Gonna ask for a look from some devs since this is a larger PR, but looks good to me now!

abandonedaccount6235 commented on misc.sk:
```diff
@@ -75,10 +75,233 @@ plants:
+	
+new music discs:
+	minecraft version = 1.13 or newer
+	11-[music ]disc¦s = minecraft:music_disc_11
```
how about `music disc 11`? (also for the other discs)

APickledWalrus replied:
Good idea! I’ll change this soon. 

I won’t be at my computer until around Tuesday so I’ll make the changes then. I was thinking maybe it could be like:

[[music]] (disc|record) 11 (double brackets b/c GitHub linking thingy)

abandonedaccount6235 replied:
you can use \ to escape that, just so you know. so:
\\[music\\] (disc|record) 11
becomes
\[music\] (disc|record) 11
or even easier you can put it in a code block:
\`\[music\] (disc|record) 11\`
becomes
`[music] (disc|record) 11`

APickledWalrus replied:
Didn’t know this, thank you!

APickledWalrus commented:
I updated music discs and updated nuggets/shulker shells to fix the conflict (nuggets were added in 1.11.1 so they can't be put with shulker shells)

Should be good to merge now.

TheBentoBox commented:
Good change. I like that the record aliases match the IDs now. 

Awesome contribution, this was a lot of work. Cheers!
